exla/c_src/exla/exla.cc

```diff
@@ -245,6 +245,49 @@ ERL_NIF_TERM deallocate_device_mem(ErlNifEnv* env, int argc, const ERL_NIF_TERM
   }
 }
 
+// Dimension Functions
+
+ERL_NIF_TERM get_dimension_size(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[]) {
+	if (argc != 2) {
+		return exla::nif::error(env, "Bad argument count.");
+	}
```

Comment:
Please adjust the spacing to 2 spaces per tab :)

Reply:
Eventually I will, in the meanwhile I'll keep this spacing -- I find it very difficult to read C/C++ code with that indentation 😊